doc/docs/master/enterprise/auth/oidc/index.md

```diff
@@ -0,0 +1,17 @@
+# Configure an OIDC Auth Provider
+
+OpenID Connect (OIDC) is an identity layer that extends authorization
+capabilities of the OAuth 2.0 security framework with authentication
+and single sign-on (SSO). Pachyderm supports authentication providers,
+such as [Keycloak](https://www.keycloak.org), that work with OIDC.
+
+Pachyderm designates port `14687` for OIDC connections. This port can
+be modified as needed. In a default testing Pachyderm deployment, the
+root URL that needs to be used to access an OIDC provider is
+`http://localhost:14687/authorization-code/callback`.
+
+
+!!! important
+    Although other identity providers might work with Pachyderm, only
```

Comment:
Okta and Auth0 have also been tested. Planning to get to Google and maybe some other ones this week.